README.md

```diff
@@ -317,17 +227,16 @@ experiment/
     ...
 ```
 
-The `Makefile` (or similar build configuration) should perform any necessary
-compilation or preprocessor steps.
-Each benchmark should expose a function (or method) called `run_iter`
-which is the entry point to the benchmark.
-In the case of compiled languages, it often makes sense to add an extra
-class to an existing benchmark, in order to provide an entry point.
+The `Makefile` should build the VMs and benchmarks needed for the experiment.
+
+Each benchmark should expose a function (or method) called `run_iter` which is
+the entry point to the benchmark. To preserve source code history, it can
+be easiest to put this function in a new file, which then imports the benchmark.
 
-The examples here uses C, Java and cPython.
-The following VMs are currently supported:
+New VMs require some support in Krun. The following VMs are currently supported
+out of the box:
 
-  * Standard Java SDK (such as Hotspot)
+  * Standard JDK (i.e. Hotspot)
```

Comment:
Some require patches :(

Reply:
Later in this section we say for a third time which platforms are supported ;)

Reply:
Right, so if some JDKs require patches, we don't support them: I think we only support HotSpot?

Reply:
Some VMs sorry. The JVM has a good enough FFI already.

There are comments at the top of the iterations runners for languages which we needed to patch for the warmup experiment. So that's:

* v8
* HHVM
* JRuby/Truffle or TruffleRuby

Reply:
OK, but I think we don't need to mention that here. I've removed the confusing stuff and been clearer that we support OpenJDK.